.github/workflows/ci.yaml

```
@@ -48,7 +48,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.6", "3.9", "3.10", "3.11"]
+        # 3.6: SLE15
+        # 3.9: RHEL/Centos/Liberty 9
+        # 3.11: Tumbleweed
+        # 3.12 Ubuntu 24.04
```

The OCD is kicking in :stuck_out_tongue: 

```suggestion
        # 3.12: Ubuntu 24.04
```